### PR TITLE
chore(tool): fix addlicense

### PR DESCRIPTION
**Pull request by universal-itengineer**
## Description

Fix [PR 114](https://github.com/deckhouse/virtualization/pull/114)
Adding copyright to files in the .github folder

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

### CODE REVIEW

File: tools/addlicense/addlicense_test.go [hunk 1]
```diff
@@ -222,3 +223,84 @@ print("Hello")
...
+	}
+
+	for _, c := range filePathCases {
+		t.Run(c.title, func(t *testing.T) {
+			resFilePathMatch := fileToCheckRe.MatchString(c.filePath)
+			require.Equal(t, c.expectedFileToCheckRe, resFilePathMatch)
+
+			license := getLicenseForFile(c.filePath)
+			require.Empty(t, license)
+			assert.NotEqual(t, CELicenseRe.MatchString(license), true)
```
Comment (diafour) [inline tools/addlicense/addlicense_test.go:303]: Use `require.NotEqual`. `assert.NotEqual` returns bool and not exits, it should be used with if:

```
if !assert.WhatEver() {
  do something.
}
```

Comment (universal-itengineer) [reply tools/addlicense/addlicense_test.go]: Got it. Done.

File: tools/addlicense/addlicense_test.go [hunk 2]
```diff
@@ -222,3 +222,83 @@ print("Hello")
...
+	}
+
+	for _, c := range filePathCases {
+		t.Run(c.title, func(t *testing.T) {
+			resFilePathMatch := fileToCheckRe.MatchString(c.filePath)
+			require.Equal(t, c.expectedFileToCheckRe, resFilePathMatch)
+
+			license := getLicenseForFile(c.filePath)
+			require.Empty(t, license)
+			require.NotEqual(t, CELicenseRe.MatchString(license), true)
```
Comment (Isteb4k) [inline tools/addlicense/addlicense_test.go:301]: require.NotEqual -> require.False

File: tools/addlicense/addlicense_test.go [hunk 3]
```diff
@@ -222,3 +222,83 @@ print("Hello")
...
+			require.Equal(t, c.expectedFileToCheckRe, resFilePathMatch)
+
+			license := getLicenseForFile(c.filePath)
+			require.NotEmpty(t, license)
+			require.Equal(t, CELicenseRe.MatchString(license), true)
+		})
+	}
+}
+
+func Test_FilesPathNoExtentionRe(t *testing.T) {
```
Comment (Isteb4k) [inline tools/addlicense/addlicense_test.go:276]: typo: Exten**T**ion -> Exten**S**ion

`Test_FilesPathNoExtensionRe`

**Review by Isteb4k — APPROVED** [review]

File: tools/addlicense/addlicense_test.go (hunk 2 above)
Comment (universal-itengineer) [reply tools/addlicense/addlicense_test.go]: fixed

**Pull request closed without merging**